t/integration/test_canvas.py

```diff
@@ -89,6 +89,18 @@ def assert_ids(r, expected_value, expected_root_id, expected_parent_id):
 class test_chord:
 
     @flaky
+    def test_group_chain(self, manager):
```

Comment:
I don't know; the `@flaky` decorator looked to be indiscriminately applied to all tests in `t/integration`, so I included it for consistency. Someone else would have to answer about whether or where it is still needed.